.gitignore

```diff
@@ -14,6 +14,8 @@ celer.egg-info
 # build
 build
 
+# virtual environment
+venv
```

Comment:
I isolated celer python environment. 
The ``9K+`` in the source control is a bit disturbing. 
So I added the ``venv`` to ``.gitignore``.

![motivation-behind-add-venv](https://user-images.githubusercontent.com/65614794/161821478-9b812527-6ba4-4b48-9cb8-5310b3aaabbd.png)

Reply:
We don't want to add one line in gitignore for each configuration of each contributor, so use `.git/info/exclude` instead